### ISSUE TITLE
[ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. 

### ISSUE THREAD

**Issue by threshold862543** [orbitdb/orbitdb#1015]
I have an IPFS daemon running and I want to initialize OrbitDB on it:

```
async function SetupIpfsAndOrbitDb (){

	//Load in IPFS and OrbitDB
	const IPFS = await import('ipfs-http-client')
	const OrbitDB = await require('orbit-db')

	//Create IPFS instance
	const ipfs = await IPFS.create()

	//Create OrbitDB instance
	const orbitdb = await OrbitDB.createInstance(ipfs)
	const newsdb = await orbitdb.docs('news')
	return newsdb
}
const odb = SetupIpfsAndOrbitDb()
```
This produces an error:
```
12|test    |     at Function.createInstance (/home/.../node_modules/orbit-db/src/OrbitDB.js:109:33)
12|test    |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
12|test    |     at async SetupIpfsAndOrbitDb (/home/.../models/forms/addnews.js:45:18) {
12|test    |   code: 'ERR_INVALID_ARG_TYPE'
12|test    | } Promise {
12|test    |   <rejected> TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Ed25519PeerIdImpl
12|test    |       at new NodeError (node:internal/errors:387:5)
12|test    |       at validateString (node:internal/validators:121:11)
12|test    |       at Object.join (node:path:1172:7)
12|test    |       at Function.createInstance (/home/.../node_modules/orbit-db/src/OrbitDB.js:109:33)
12|test    |       at processTicksAndRejections (node:internal/process/task_queues:96:5)
12|test    |       at async SetupIpfsAndOrbitDb (/home/.../models/forms/addnews.js:45:18) {
12|test    |     code: 'ERR_INVALID_ARG_TYPE'
12|test    |   }
12|test    | }
```
I'm not sure what the problem is.

**Comment by Codefa** [comment]
I have the same issue, we can fix it by downgrading `orbitdb` version
to
`"orbit-db": "^0.26.1"` or  `"orbit-db": "^0.24.1"`

**Comment by threshhold862543** [comment]
It was also fixed in #1004.

**Comment by haadcode** [comment]
This should be fixed in 1.0.0. Note the [new package name](https://www.npmjs.com/package/@orbitdb/core). Note also that `ipfs-http-client` is not supported in OrbitDB anymore.

If the issue persists, feel free to re-open the issue.
